Source/Infrastructure/Integration/Kysect.Shreks.Integration.Github/Invites/GithubInvitingWorker.cs

```diff
@@ -117,6 +119,49 @@ private async Task ProcessSubject(IMediator mediator, GithubSubjectCourseAssocia
         }
     }
 
+    private async Task GenerateRepositoryForAdded(InviteResult inviteResult, string organizationName)
+    {
+        GitHubClient client = await _clientProvider.GetClient(organizationName);
+        IReadOnlyList<Repository>? repositories = await client.Repository.GetAllForOrg(organizationName);
+        Repository? templateRepository = repositories.FirstOrDefault(r => r.IsTemplate);
+        const string missingRepoException = "No template repository found for organization {0}";
+
+        if (templateRepository is null)
+        {
+            throw new InfrastructureInvalidOperationException(string.Format(missingRepoException, organizationName));
```

Comment:
Вообще я имел в виду что-то типа:

```suggestion
        if (templateRepository is null)
        {
            string message = $"No template repository found for organization {organizationName}";
            throw new InfrastructureInvalidOperationException(message);
```